packs/promtail/templates/promtail.nomad.tpl

```diff
@@ -0,0 +1,101 @@
+job [[ template "job_name" . ]] {
+
+  [[ template "region" . ]]
+
+  datacenters = [ [[ range $idx, $dc := .promtail.datacenters ]][[if $idx]],[[end]][[ $dc | quote ]][[ end ]] ]
+  type = "system"
+  
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
```

Comment:
Could this be added as a default to a variable like the [Prometheus pack](https://github.com/hashicorp/nomad-pack-community-registry/blob/main/packs/prometheus/variables.hcl#L25-L39) does and then use this within the template? It would allow non-linux user to alter this and run it with modified variables.